cmd/edit/externalauthproviders/cmd.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externalauthproviders
+
+import (
+	"os"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	name                             string
+	issuerName                       string
+	issuerAudiences                  []string
+	issuerUrl                        string
+	issuerCaFile                     string
+	claimMappingGroupsClaim          string
+	claimMappingUsernameClaim        string
+	claimValidationRuleClaim         string
+	claimValidationRuleRequiredValue string
```

Comment:
The args are the same for both create and edit. I would suggest moving these to a `pkg` to be shared between cmd's